angular/core/components/email_settings_page/email_settings_page_controller.coffee

```diff
@@ -1,7 +1,20 @@
-angular.module('loomioApp').controller 'EmailSettingsPageController', (Records, FormService, CurrentUser, $location) ->
+angular.module('loomioApp').controller 'EmailSettingsPageController', (Records, FormService, CurrentUser, $location, ModalService, ChangeVolumeForm) ->
 
   @user = CurrentUser.clone()
 
+  @groupVolume = (group) ->
+    group.membershipFor(CurrentUser).volume()
+
+  @changeDefaultMembershipVolume = ->
+    ModalService.open ChangeVolumeForm,
+      model: => (_.first CurrentUser.groups()).membershipFor(CurrentUser),
+      setDefault: => true
```

Comment:
The more I see this, the more I want to make this more polymorphic, so that we can pass the CurrentUser for defaultMembershipVolume, the Group for group volume, and a Discussion for discussion volume.

I think that'll eliminate the need for this setDefault boolean, and this pretty-awkward 'get the membership of the current user's first group'  call

:grey_question: